src/index.html

```diff
@@ -84,23 +85,264 @@ <h2 class="hero__social-media-links__title">Let&apos;s connect</h2>
+              sign-ups required.
+            </p>
+
+            <!-- Project Tech Stack -->
+            <ul class="works__project-card__tech-stack-list">
+              <li class="works__project-card__tech-stack-item">HTML</li>
+              <li class="works__project-card__tech-stack-item">CSS</li>
+              <li class="works__project-card__tech-stack-item">JavaScript</li>
+            </ul>
+
+            <!-- See Project Details Button -->
+            <button class="primary-button">See project</button>
+          </li>
+        </ul>
       </section>
 
       <!-- About Me -->
-      <section class="container">
+      <section class="about-me container">
+        <h2 class="about-me__header">About Me</h2>
+        <p class="paragraph-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
         <!-- Social Media Links -->
+        <div class="social-media-links">
+          <h2 class="social-media-links__title">Let&apos;s connect</h2>
+          <ul class="social-media-links__icons">
+            <!-- GitHub -->
+            <li>
+              <a href="https://github.com/henokkhm">
+                <img src="assets/icons/github.svg" alt="GitHub icon" />
+              </a>
+            </li>
+
+            <!-- LinkedIn -->
+            <li>
+              <a href="https://www.linkedin.com/in/henokkhm/">
+                <img src="assets/icons/linked-in.svg" alt="linkedIn icon" />
+              </a>
+            </li>
+
+            <!-- Other -->
+            <li>
+              <a href="#">
+                <!-- TODO: What social media has this icon? -->
+                <img src="assets/icons/unknown.svg" alt="Social media icon" />
+              </a>
+            </li>
+
+            <!-- Twitter -->
+            <li>
+              <a href="https://twitter.com/henokkhm">
+                <img src="assets/icons/twitter.svg" alt="Twitter icon" />
+              </a>
+            </li>
+
+            <!-- Medium -->
+            <li>
+              <a href="#">
+                <!-- TODO: Get link to Medium -->
+                <img src="assets/icons/medium.svg" alt="Medium icon" />
+              </a>
+            </li>
+          </ul>
+        </div>
+
+        <!-- Get Resume Button -->
+        <button class="primary-button">Get my resume</button>
+
+        <!-- About me summary -->
+        <ul class="about-me__summary">
+          <li>
+            <div class="about-me__summary__header">
+              <span>Languages</span>
+              <img src="assets/icons/caret.svg" alt="expand-button" />
+            </div>
+            <ul class="about-me__languages-list">
+              <li class="about-me__language-item">
+                <img src="assets/icons/js.png" alt="JavaScript icon" />
```

Comment:
According to the Figma template, kindly remove the gray line at the top of the first `about-me__language-item`.

<img width="357" alt="Screenshot 2023-04-28 at 07 10 38" src="https://user-images.githubusercontent.com/91237525/235134982-201e5e9b-ddc6-4a37-8e11-5672094c9754.png">